packages/loader/container-loader/README.md

```diff
@@ -173,7 +173,7 @@ Errors are of [ICriticalContainerError](../../common/container-definitions/src/e
 There are 4 sources of errors:
 
 1. [ContainerErrorType](../../common/container-definitions/src/error.ts) - errors & warnings raised at loader level
```

Comment:
I think these other links need to have their text changed too, since we renamed them as well.